Registry/Registry.EM_COMMON

```diff
@@ -1649,47 +1649,47 @@ state    integer  landuse_luseas  -      misc       -          -     r
 state    integer  landuse_isn     -      misc       -          -     r
 state    real     lu_state        p      misc       -          -     r
 
-i1       real  th_phy          ikj      misc        1         -
-i1       real  pi_phy          ikj      misc        1         -
-i1       real  p_phy           ikj      misc        1         -
+state    real  th_phy          ikj      misc        1         -      -      "TH_PHY"                       "Potential Temperature"         "K"
+state    real  pi_phy          ikj      misc        1         -      -      "PI_PHY"                       "Exner function"         "dimensionless"
+state    real  p_phy           ikj      misc        1         -      -      "P_PHY"                        "Pressure"   "Pa"
 state    real  t_phy           ikj      misc        1         -      r      "T_PHY"                        "Temperature"         "K"
 state    real  u_phy           ikj      misc        1         -      r      "U_PHY"                        "x-wind component at mass point"         "m s-1"
 state    real  v_phy           ikj      misc        1         -      r      "V_PHY"                        "y-wind component at mass point"         "m s-1"
-i1       real  dz8w            ikj      misc        1         Z
-i1       real  p8w             ikj      misc        1         Z
-i1       real  t8w             ikj      misc        1         Z
+state    real  dz8w            ikj      misc        1         Z      -      "DZ8W"                         "d(z)/d(eta), full levels"  "m"
+state    real  p8w             ikj      misc        1         Z      -      "P8W"                          "Pressure, full levels"  "Pa"
+state    real  t8w             ikj      misc        1         Z      -      "T8W"                          "Temperature, full levels"  "K"
 i1       real  rho_phy         ikj      misc        1         -
-i1    logical  CU_ACT_FLAG     ij       misc        1         -
+state logical  CU_ACT_FLAG     ij       misc        1         -      -      "CU_ACT_FLAG"                  "Some great mystery"   "unknown"
```

Comment:
@dudhia @weiwangncar
Folks,
Any suggestions for the description and units fot the cu_act_flag?